webrisk/README.md

```diff
@@ -0,0 +1,87 @@
+# Reference Implementation for the Usage of Google Cloud WebRisk APIs (Beta)
+
+The `webrisk` Go package can be used with the
+[Google Cloud WebRisk APIs (Beta)](https://cloud.google.com/web-risk/)
+to access the Google Cloud WebRisk lists of unsafe web resources. Inside the
+`cmd` sub-directory, you can find two programs: `wrlookup` and `wrserver`. The
+`wrserver` program creates a proxy local server to check URLs and a URL
+redirector to redirect users to a warning page for unsafe URLs. The `wrlookup`
+program is a command line service that can also be used to check URLs.
+
+This **README.md** is a quickstart guide on how to build, deploy, and use the
+`safebrowsing` Go package. It can be used out-of-the-box. The GoDoc and API
+documentation provide more details on fine tuning the parameters if desired.
+
+
+# How to Build
+
+To download and install from the source, run the following command:
+
+```
+go get github.com/JJLionHeart/golang-samples/webrisk
```

Comment:
Probably want the GoogleCloudPlatform version?

Reply:
Yes, I pushed this change to the master branch by mistake, it should now contains the GoogleCloutPlatform version on master